test/packer_oci_example_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/oci"
+	"github.com/gruntwork-io/terratest/modules/packer"
+)
+
+const compartmentOcid = "ocid1.compartment.oc1..aaaaaaaa"
+
+// An example of how to test the Packer template in examples/packer-basic-example using Terratest.
+func TestPackerOciExample(t *testing.T) {
+	t.Parallel()
+
+	baseImageOcid := oci.BaseImageOcid(t, compartmentOcid, "Canonical Ubuntu", "18.04")
+
+	packerOptions := &packer.Options{
+		// The path to where the Packer template is located
+		Template: "../examples/packer-basic-example/build.json",
+
+		// Variables to pass to our Packer build using -var options
+		Vars: map[string]string{
+			"oci_base_image_ocid":     baseImageOcid,
+			"oci_compartment_ocid":    compartmentOcid,
+			"oci_availability_domain": "YHQr:PHX-AD-1",
+			"oci_subnet_ocid":         "ocid1.subnet.oc1.phx.aaaaaaaa",
```

Comment:
What are these availability domains and subnet IDs? Are these unique to your account?

Reply:
_Availability domains_ are specific per _Region_ and are already created by the cloud provider. _Subnet ID_ is specific per account and has to be created by user. Both are [mandatory parameters](https://www.packer.io/docs/builders/oracle-oci.html#required) of OCI _Packer_ builder. Similarly to my previous comment, I can investigate their externalization from the source code.